app/serializers/users.js

```diff
@@ -3,6 +3,18 @@ const { pick } = require('lodash');
 const { underscoreKeys } = require('../helpers/object_utils');
 const { pageSerializer } = require('./pagination');
 
-exports.showUserSerializer = user => underscoreKeys(pick(user, 'id', 'firstName', 'lastName', 'email'));
+const jobPosition = ({ points }) => {
+  if (points <= 5) return 'DEVELOPER';
+  if (points <= 9) return 'LEAD';
+  if (points <= 19) return 'TL';
+  if (points <= 29) return 'EM';
+  if (points <= 49) return 'HEAD';
+  return 'CEO';
+};
```

Comment:
This could be in another place, so it can be use by other parts of the api. Where do you think it could be?